### PR TITLE
Version 0.1.0

### PR DESCRIPTION
**Pull request by daniel112**
## Context
* releasing a new version for Sessions component
* fix minor issues with styling when changing width sizes

## **Update:**
    - keg core update 5.0.1
    - tap update 0.1.0
    - imported the .css file in root index.js
    - removed the mock css import from dynamicClassMap
    - added theme.config import in the App's entry point

## Testing
* **Build**: `keg evf pack run package=docker.pkg.github.com/simpleviewinc/keg-packages/tap:version-0.1.0 command=test:build` 
        * navigate to http://local.kegdev.xyz/
        * verify that styling adjust accordingly when changing to/from mobile sizes (480px)

* **Tap**: `keg evf pack run package=docker.pkg.github.com/simpleviewinc/keg-packages/tap:version-0.1.0 command=web` 
    * navigate to http://local.kegdev.xyz/
    * verify that styling adjust accordingly when changing to/from mobile sizes (480px)

* **Tap unit test**: `keg evf pack run package=docker.pkg.github.com/simpleviewinc/keg-packages/tap:version-0.1.0 command=test` 
    * verify all test passes

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,5 +1,8 @@
+/**
+ * NOTE: any changes made on this file should exists in apps/Sessions.js if you want access to it
```
Comment (mpcarolin) [inline index.js:2]: really minor, but this should say "if you want access to it in the Sessions export"

File: src/theme/dynamicClassMap.js [hunk 2]
```diff
@@ -11,7 +11,6 @@ import { styleSheetParser } from '@keg-hub/re-theme/styleParser'
 
 // Test styles, to ensure the styleSheetParser is working properly
 // They will be removed once we get some real style sheets from Events Force
```
Comment (mpcarolin) [inline src/theme/dynamicClassMap.js:13]: I think the comments for this should be removed too

**Review by mpcarolin — COMMENTED** [review]
Passed all testing 👍 

I have a couple of minor suggestions

File: src/theme/dynamicClassMap.js (hunk 2 above)
Comment (lancetipton) [reply src/theme/dynamicClassMap.js:13]: Yah, I agree

**Review by lancetipton — APPROVED** [review]
Looks good. Nothing blocking, but there are some minor notes updates that could be made

**Pull request merged**